imod/tests/test_mf6/test_mf6_npf.py

```diff
@@ -1,13 +1,15 @@
 import pathlib
 import re
 import textwrap
+from copy import deepcopy
 
 import numpy as np
 import pytest
 import xarray as xr
 
 import imod
 from imod.schemata import ValidationError
+from imod.tests.test_mf6.test_mf6_dis import _load_imod5_data_in_memory
```

Comment:
I think we can load the data in memory in the fixture immediately, this will save some run time and avoids that we have to import helper functions from strange places. You can move the helper function to the fixture location.

Reply:
done